gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -68,6 +88,11 @@ p {
   display: none;
 }
 
+.table td, table th{
```

Comment:
If we have eliminated all tables from being rendered, we can rid facultyprofile.css of selectors pertaining to tables (not just this selector, but any others as well).